parrot/setup_kde.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+sudo apt install parrot-kde sddm 
+
+sudo systemctl enable sddm.service
+
+sudo cp -r -v Dotfiles/.config/* ~/.config/
+sudo cp -r -v Dotfiles/.local/* ~/.local/
```

Comment:
Same, no sudo

Reply:
where's the fun without sudo though

Reply:
Sudo is fun but it ownership goes ro root right?